File: Task9.1p/app/src/main/java/com/example/lostandfoundapp/MainActivity.java
package com.example.lostandfoundapp;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        Button buttonNewAdd = findViewById(R.id.create_new_ad);
        Button buttonShowAll = findViewById(R.id.show_all_ads);
        Button buttonshowmap = findViewById(R.id.show_on_map);
        buttonNewAdd.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(MainActivity.this, NewAdd.class);
                startActivity(intent);
            }
        });

        buttonShowAll.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(MainActivity.this, ShowAllAddspage.class);
                startActivity(intent);
            }
        });
        buttonshowmap.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent i = new Intent(MainActivity.this, ShowOnMap.class);
                    startActivity(i);
            }
        });
    }
}

File: Task5.1c/NewsAPP/app/src/main/java/com/example/newsapp/MainActivity.kt
package com.example.newsapp

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import androidx.fragment.app.Fragment
import com.example.newsapp.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity() {
    private lateinit var binding: ActivityMainBinding;
    val fragment: Fragment? = null
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityMainBinding.inflate(layoutInflater)
        setContentView(binding.root)
        binding.apply {
            showFragment(NewMainPage())
        }
    }


    fun showFragment(fragment: Fragment){
        supportFragmentManager.beginTransaction().replace(R.id.Mainframelayout,fragment).commit()
    }
}